src/components/Layout/index.scss

```diff
@@ -456,6 +456,10 @@ ol {
   }
 }
 
+a code {
+  @extend .underline;
+}
+
```

Comment:
Please move this to a separate PR; we can get it merged independently then :+1:

Reply:
I think there weren't any links on code yet (or did I just not see them?), so this was really for the parts I've written.

Reply:
OK